warp/Network/Wai/Handler/Warp/FileInfoCache.hs

```diff
@@ -7,14 +7,14 @@ module Network.Wai.Handler.Warp.FileInfoCache (
     getInfo, -- test purpose only
 ) where
 
+import Control.Exception (bracket, onException, throwIO)
```

Comment:
`UnliftIO.onException` also runs with `uninterruptibleMask`. Making sure async exceptions wait until after the cleanup is done.